src/renderer/widgets/CreateWallet/ui/MultisigWallet/NameNetworkSelection.tsx

```diff
@@ -0,0 +1,89 @@
+import { useUnit } from 'effector-react';
+import { useForm } from 'effector-forms';
+import { FormEvent } from 'react';
+
+import { Button, FootnoteText, Input, InputHint, Select, SmallTitleText } from '@shared/ui';
+import { useI18n } from '@app/providers';
+import { DropdownOption } from '@shared/ui/types';
+import { networkModel, networkUtils } from '@entities/network';
+import { ChainTitle } from '@entities/chain';
+import { type Chain } from '@shared/core';
+import { formModel } from '../../model/form-model';
+import { flowModel } from '../../model/flow-model';
+import { Step } from '../../lib/types';
+
+const getChainOptions = (chains: Chain[]): DropdownOption<Chain>[] => {
+  return chains
+    .filter((c) => networkUtils.isMultisigSupported(c.options))
+    .map((chain) => ({
+      id: chain.chainId.toString(),
+      element: <ChainTitle chain={chain} />,
+      value: chain,
+    }));
+};
+
+export const NameNetworkSelection = () => {
+  const { t } = useI18n();
+
+  const chains = useUnit(networkModel.$chains);
+  const {
+    fields: { name, chain },
+    submit,
+  } = useForm(formModel.$createMultisigForm);
+
+  const chainOptions = getChainOptions(Object.values(chains));
+  const canContinue = name.isValid;
```

Comment:
This variable is probably not worth defining for a single use case `disabled={!canContinue}`